dashboard/app/models/levels/level.rb

```diff
@@ -241,9 +241,6 @@ def load_level_xml(xml_node)
         hash['notes'] = Encryption.decrypt_object(encrypted_notes)
       end
     rescue Encryption::KeyMissingError
-      # developers and adhoc environments must be able to seed levels without properties_encryption_key
-      non_ci_test = rack_env == :test && !CDO.ci && !CDO.chef_managed
-      raise unless rack_env?(:development) || rack_env?(:adhoc) || non_ci_test
```

Comment:
I'm feeling conflicted. It does seem useful for us to have something in place to ensure that if we somehow lose access to the key in production (or ideally also in the DTT) we get a highly visible error at an early point in the build process, but I don't particularly want to increase the discrepancy between prod and our other environments by putting a `raise if rack_env?(:production)` here, and the existing logic to distinguish between the actual DTT and other test scenarios is definitely gnarly.

Is there somewhere else in the codebase where it would be more appropriate to have environment-specific logic for handling this edge case? Or do you have an argument for why we don't need to worry about this happening in prod at all?